PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx

```diff
@@ -111,6 +114,10 @@ end
 
 function MasterControl:SetIsJumping(jumping)
 	isJumping = jumping
+	local humanoid = self:GetHumanoid()
+	if humanoid and not humanoid.PlatformStand then
+		humanoid.Jump = isJumping
```

Comment:
Is there any reason we need to set the jump here rather than letting the renderstep function update it?